ts.js

```diff
@@ -1,12 +1,29 @@
-module.exports = {
-  parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
-  extends: [
-    './eslint.js',
-    'plugin:@typescript-eslint/recommended'
-  ],
+import tseslint from 'typescript-eslint';
+import nodePlugin from 'eslint-plugin-n';
+import eslint from './eslint';
+
+const nodeConfig = {
   rules: {
     'n/no-unsupported-features/es-syntax': ['error', { 'ignores': ['modules'] }],
     'n/no-missing-import': ['error', { 'tryExtensions': ['.js', '.ts'] }]
-  }
-};
+  },
+}
+
+const tslintConfig = tseslint.config({
+  plugins: {
+    '@typescript-eslint': tseslint.plugin,
+  },
+  languageOptions: {
+    parser: tseslint.parser,
+    parserOptions: {
+      project: true,
+    },
+  },
+});
+
+export default [
+  eslint,
+  ...nodePlugin.configs["flat/mixed-esm-and-cjs"],
+  tslintConfig,
```

Comment:
this maybe a wrong usage. In my test, `tslintconfig` it's not loaded.
![inspect](https://github.com/user-attachments/assets/e2d114b6-2093-4a16-bc85-a67428cd659a)

Reply:
perhaps fixed...

Reply:
Maybe I was wrong before and it might be okay. 

But I don't know why not using ·tseslint.configs.recommended·
https://typescript-eslint.io/users/configs/#getting-started